lib/opentok.js

```diff
@@ -115,7 +115,11 @@ OpenTok = function (apiKey, apiSecret, env) {
       clientConfig.uaAddendum = env.uaAddendum;
       apiConfig.uaAddendum = env.uaAddendum;
     }
-  }
+
+    if (parseInt(env.timeout, 10)) {
```

Comment:
I believe we need to validate if `env.timeout` exists before we can validate it as an int.  This might fail if there is no timeout specified. What do you think?

Reply:
No, I don't think we need to validate timeout. env is validated in the `else if`, and if `timeout` doesn't exist on env, `parseInt(undefined/null)` will return `NaN`, which is falsey.

Reply:
You are right, good catch!